docs/signUp/signup.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login | ShoeStore</title>
+    <link rel="stylesheet" href="../../css/main.css">
```

Comment:
you can keep the main CSS file at the end since you'll be overriding the styles

Reply:
oh sure ..thanks .